src/main/java/com/github/tomakehurst/wiremock/http/StubResponseRenderer.java

```diff
@@ -49,6 +50,8 @@ public Response render(ResponseDefinition responseDefinition) {
 		}
 
 		Response response = buildResponse(responseDefinition);
+		LinkedList<ResponseTransformer> responseTransformers = new LinkedList<>(this.responseTransformers);
```

Comment:
I use the out-of-fashion `LinkedList` here because of the recursion of the `applyTransformations`.